test/jdk/java/nio/channels/Selector/SelectWithConsumer.java

```diff
@@ -354,12 +352,10 @@ public void testWakeupDuringSelect() throws Exception {
         // select(Consumer, timeout)
         try (Selector sel = Selector.open()) {
             scheduleWakeup(sel, 1, SECONDS);
-            long start = System.currentTimeMillis();
+            long start = millisTime();
             int n = sel.select(k -> assertTrue(false), 60*1000);
-            long duration = System.currentTimeMillis() - start;
+            expectDuration(start, 501, 10*1000 - 1);
```

Comment:
I think I would make this one 0, 20_000, again for slow machines and also because the test is about select rather than the accuracy of scheduleWakeup.